guides/installation.md

```diff
@@ -1,3 +1,3 @@
 # Installation
 
-In order to install Pyegi, you may clone it from its [Github page](https://github.com/SSgumS/Pyegi), extract the zipped file and using windows PowerShell, run the file "install.bat" from the extracted files. Right now, the package is available for Windows operating systems.
+In order to install Pyegi, you may clone it from its [Github page](https://github.com/SSgumS/Pyegi), extract the zipped file and using windows PowerShell, run the file `install.bat` from the extracted files. Although the installation process is currently only available for Windows, the Pyegi itself is cross-platform compatible, so if it is installed correctly, it should run on other platforms as well.
```

Comment:
that "installed correctly" is a big if. they need to know the whole process of installation in Windows and replace the steps they need for their OS

Reply:
still...
i would go with "if installed manually" or sth like this

Reply:
fixed